examples/sandbox/package.json

```diff
@@ -23,7 +23,8 @@
     "debug": "^4.1.1",
     "jquery": "^3.4.1",
     "json": "^9.0.6",
-    "parcel-bundler": "^1.12.4"
+    "parcel-bundler": "^1.12.4",
+    "secp256k1": "^4.0.2"
```

Comment:
Sandbox fixed? I thought we tried that, but good to see it worked. Did you ever verify that the package is a dependency in the binanceSignTx package?

Reply:
not fixed :(

Reply:
it was atom that this actually broke on. and deff the package its complaining of is already in cosmos-tx-builder ```{
  "name": "cosmos-tx-builder",
  "version": "6.3.3",
  "scripts": {
    "test": "jest"
  },
  "dependencies": {
    "bech32": "^1.1.4",
    "bip32": "^2.0.5",
    "bip39": "^3.0.2",
    "bitcoinjs-lib": "^5.1.10",
    "coininfo": "^5.1.0",
    "crypto-js": "^3.3.0",
    "dotenv": "^8.2.0",
    "secp256k1": "^3.8.0"
  }
}

Reply:
ill go ahead and remove as this didnt help anything